samples/ambient-argo/application/details.yaml

```diff
@@ -0,0 +1,50 @@
+##################################################################################################
+# Details service
+##################################################################################################
+apiVersion: v1
+kind: Service
+metadata:
+  name: details
+  labels:
+    app: details
+    service: details
+spec:
+  ports:
+  - port: 9080
+    name: http
+  selector:
+    app: details
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: bookinfo-details
+  labels:
+    account: details
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: details-v1
+  labels:
+    app: details
+    version: v1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: details
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: details
+        version: v1
+    spec:
+      serviceAccountName: bookinfo-details
+      containers:
+      - name: details
+        image: docker.io/istio/examples-bookinfo-details-v1:1.18.0
```

Comment:
nit: this will probably not be updated when the rest of bookinfo is. Via samples/bookinfo/src/build-services.sh.

Ideally we symlink or something, but I don't see how that would work given the setup we have. I don't mind if we do nothing, either.

Reply:
I mentioned the same in another file. I figure we can just add another set of files to `sed`.